ui/analyse/src/study/studyForm.ts

```diff
@@ -294,7 +294,7 @@ export function view(ctrl: StudyFormCtrl): VNode {
               'submit',
               _ =>
                 isNew ||
-                prompt(ctrl.trans.noarg('deleteTheEntireStudy') + ' ' + 'Type the name of the study to confirm') ===
+                prompt(ctrl.trans.noarg('deleteTheEntireStudy') + ' ' + ctrl.trans.noarg('retypeStudyName')) ===
```

Comment:
Just create a new translation key for the combined new sentence. `deleteTheEntireStudy` isn't used anywhere else and this allows translators to understand the context better and translate everything into a coherent whole.

Reply:
Is it safe to remove `deleteTheEntireStudy` immediately? I've left it in translation/source/study.xml for now.